src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

```diff
@@ -912,19 +912,37 @@ public void Start(string[] args)
         /// they will be stopped first. The DependentServices property lists this set
         /// of services.
         /// </summary>
-        public unsafe void Stop()
+        public void Stop()
+        {
+            Stop(true);
```

Comment:
```suggestion
            Stop(stopDependentServices: true);
```

Reply:
Nice to see that I'm not the only one who doesn't like unnamed boolean parameters. 😄 
In this case I omitted the parameter name because it is specified just a few lines below, but I agree that a bit of extra clarity doesn't hurt :)